supervision/detection/utils.py

```diff
@@ -447,6 +447,62 @@ def xyxy_to_xcycarh(xyxy: np.ndarray) -> np.ndarray:
     return result.astype(float)
 
 
+def normalized_xyxy_to_absolute_xyxy(
```

Comment:
please rename this function to `denormalize_boxes` to match existing `clip_boxes`, `pad_boxes` and `scale_boxes`